### ISSUE TITLE
Dry Run

### ISSUE THREAD

**Issue by avzero07** [avzero07/dl-based-sensor-calib#10]
Using the model developed in #9 do a dry-run train / test job.

**Comment by avzero07** [comment]
<pre>
Num = 1109
Len = 1109
Test Set Average Loss 0.2321
</pre>

<pre>
Start Inference
Num = 742
Len = 742
Test Set Average Loss 0.0517
</pre>

10 Epochs 

<pre>
Start Training
149it [02:09,  1.15it/s]
0it [00:00, ?it/s]
Train Epoch: 0	Mean Loss: 0.000790
149it [02:09,  1.15it/s]
0it [00:00, ?it/s]
Train Epoch: 1	Mean Loss: 0.000701
149it [02:11,  1.13it/s]
0it [00:00, ?it/s]
Train Epoch: 2	Mean Loss: 0.000589
149it [02:11,  1.14it/s]
0it [00:00, ?it/s]
Train Epoch: 3	Mean Loss: 0.000560
149it [02:10,  1.14it/s]
0it [00:00, ?it/s]
Train Epoch: 4	Mean Loss: 0.000504
149it [02:09,  1.15it/s]
0it [00:00, ?it/s]
Train Epoch: 5	Mean Loss: 0.000464
149it [02:08,  1.16it/s]
0it [00:00, ?it/s]
Train Epoch: 6	Mean Loss: 0.000450
149it [02:08,  1.16it/s]
0it [00:00, ?it/s]
Train Epoch: 7	Mean Loss: 0.000423
149it [02:08,  1.16it/s]
0it [00:00, ?it/s]
Train Epoch: 8	Mean Loss: 0.000398
149it [02:07,  1.17it/s]
Train Epoch: 9	Mean Loss: 0.000359
</pre>

**Comment by avzero07** [comment]
The crude model overfits, but that isn't the way to go.

<pre>
Num = 1109
Len = 1109
Test Set Average Loss 0.2050
</pre>

<pre>
Start Inference
Num = 742
Len = 742
Test Set Average Loss 0.0215
</pre>

<pre>
0it [00:00, ?it/s]
Start Training
149it [02:05,  1.19it/s]
0it [00:00, ?it/s]
Train Epoch: 0	Mean Loss: 0.001771
149it [02:05,  1.19it/s]
0it [00:00, ?it/s]
Train Epoch: 1	Mean Loss: 0.000786
149it [02:08,  1.16it/s]
0it [00:00, ?it/s]
Train Epoch: 2	Mean Loss: 0.000582
149it [02:11,  1.14it/s]
0it [00:00, ?it/s]
Train Epoch: 3	Mean Loss: 0.000475
149it [02:07,  1.17it/s]
0it [00:00, ?it/s]
Train Epoch: 4	Mean Loss: 0.000402
149it [02:05,  1.19it/s]
0it [00:00, ?it/s]
Train Epoch: 5	Mean Loss: 0.000356
149it [02:08,  1.16it/s]
0it [00:00, ?it/s]
Train Epoch: 6	Mean Loss: 0.000319
149it [02:07,  1.17it/s]
0it [00:00, ?it/s]
Train Epoch: 7	Mean Loss: 0.000287
149it [02:07,  1.17it/s]
0it [00:00, ?it/s]
Train Epoch: 8	Mean Loss: 0.000274
149it [02:07,  1.17it/s]
0it [00:00, ?it/s]
Train Epoch: 9	Mean Loss: 0.000239
149it [02:07,  1.17it/s]
0it [00:00, ?it/s]
Train Epoch: 10	Mean Loss: 0.000228
149it [02:07,  1.17it/s]
0it [00:00, ?it/s]
Train Epoch: 11	Mean Loss: 0.000214
149it [02:08,  1.16it/s]
0it [00:00, ?it/s]
Train Epoch: 12	Mean Loss: 0.000198
149it [02:11,  1.13it/s]
0it [00:00, ?it/s]
Train Epoch: 13	Mean Loss: 0.000194
149it [02:11,  1.13it/s]
0it [00:00, ?it/s]
Train Epoch: 14	Mean Loss: 0.000179
149it [02:11,  1.13it/s]
0it [00:00, ?it/s]
Train Epoch: 15	Mean Loss: 0.000170
149it [02:11,  1.13it/s]
0it [00:00, ?it/s]
Train Epoch: 16	Mean Loss: 0.000158
149it [02:11,  1.13it/s]
0it [00:00, ?it/s]
Train Epoch: 17	Mean Loss: 0.000146
149it [02:11,  1.14it/s]
0it [00:00, ?it/s]
Train Epoch: 18	Mean Loss: 0.000145
149it [02:10,  1.14it/s]
0it [00:00, ?it/s]
Train Epoch: 19	Mean Loss: 0.000138
149it [02:11,  1.14it/s]
0it [00:00, ?it/s]
Train Epoch: 20	Mean Loss: 0.000133
149it [02:11,  1.14it/s]
0it [00:00, ?it/s]
Train Epoch: 21	Mean Loss: 0.000129
149it [02:10,  1.14it/s]
0it [00:00, ?it/s]
Train Epoch: 22	Mean Loss: 0.000123
149it [02:10,  1.14it/s]
0it [00:00, ?it/s]
Train Epoch: 23	Mean Loss: 0.000120
149it [02:03,  1.20it/s]
0it [00:00, ?it/s]
Train Epoch: 24	Mean Loss: 0.000114
149it [02:06,  1.18it/s]
0it [00:00, ?it/s]
Train Epoch: 25	Mean Loss: 0.000108
149it [02:06,  1.17it/s]
0it [00:00, ?it/s]
Train Epoch: 26	Mean Loss: 0.000110
149it [02:05,  1.19it/s]
0it [00:00, ?it/s]
Train Epoch: 27	Mean Loss: 0.000102
149it [02:05,  1.19it/s]
0it [00:00, ?it/s]
Train Epoch: 28	Mean Loss: 0.000098
149it [02:05,  1.19it/s]
0it [00:00, ?it/s]
Train Epoch: 29	Mean Loss: 0.000096
149it [02:05,  1.18it/s]
0it [00:00, ?it/s]
Train Epoch: 30	Mean Loss: 0.000096
149it [02:05,  1.19it/s]
0it [00:00, ?it/s]
Train Epoch: 31	Mean Loss: 0.000092
</pre>
